sentry_sdk/integrations/gcp.py

```diff
@@ -159,22 +159,22 @@ def _get_google_cloud_logs_url(initial_time):
     Returns:
         str -- Google Cloud Logs Console URL to logs.
     """
-    hour_ago = initial_time - timedelta(hours=1)
+    hour_ago = final_time - timedelta(hours=1)
 
     url = (
         "https://console.cloud.google.com/logs/viewer?project={project}&resource=cloud_function"
         "%2Ffunction_name%2F{function_name}%2Fregion%2F{region}&minLogLevel=0&expandAll=false"
-        "&timestamp={initial_time}&customFacets=&limitCustomFacetWidth=true"
-        "&dateRangeStart={timestamp_start}&dateRangeEnd={timestamp_end}"
-        "&interval=PT1H&scrollTimestamp={timestamp_current}"
+        "&timestamp={initial_time}Z&customFacets=&limitCustomFacetWidth=true"
+        "&dateRangeStart={timestamp_start}Z&dateRangeEnd={timestamp_end}Z"
+        "&interval=PT1H&scrollTimestamp={timestamp_current}Z"
```

Comment:
I'd expect the timestamps to be explicitly formatted to RFC3399 format as in https://github.com/getsentry/sentry-python/pull/797.

There should not be several `Z`s manually placed in the constructed URL string, that's very error prone and repetitive.

The other thing here is that we're implicitly relying on how a datetime object is converted to string... Unlike RFC3399, it is missing the `T` between date and time segments, and has instead a space. I wonder if that's intentional or just an accident.

Reply:
Probably the Sentry UI will not handle links correctly if the URLs contain spaces. (If there had to be any actual spaces, they should be URL-encoded as `%20`.)

Reply:
It was by accident as currently we are relying on the time format given by datetime object.
I'll make this change with appropriate string format.

Reply:
Thank you!